### PR TITLE
Add support for Go modules

### PR DESCRIPTION
**Pull request by tklauser**
Signed-off-by: Tobias Klauser <tklauser@distanz.ch>

### CODE REVIEW

**Comment by vishvananda** [comment]
This lgtm. @aboch any reason we shouldn't merge this. Seems like merging and then shipping a 1.1.0 release that supports modules is a good idea.

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module github.com/vishvananda/netlink
+
+go 1.12
+
+require (
+	github.com/vishvananda/netns v0.0.0-20180720170159-13995c7128cc
```
Comment (thaJeztah) [inline go.mod]: If this PR looks good to you @vishvananda @aboch, it's probably nice to have https://github.com/vishvananda/netns/pull/35 merged before this one (and, if possible, tag a release for https://github.com/vishvananda/netns)

**Comment by tklauser** [comment]
Ping?

**Comment by jellonek** [comment]
@aboch ping

**Comment by aboch** [comment]
LGTM

File: go.mod (hunk 1 above)
Comment (aboch) [reply go.mod]: @tklauser the netns PR is merged. You may want to update this reference then.

Comment (tklauser) [reply go.mod]: Done, thanks.

**Pull request merged**